### PR TITLE
Minor docs changes for 1.0 messaging

### PR DESCRIPTION
**Pull request by hacktivist123**
Fixes: #1672

### CODE REVIEW

File: docs/content/en/docs/overview/_index.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ linkTitle: "Overview"
 weight: 1
 icon: "overview"
 description: >
-  Discover Tetragon and its capabilities
+  Discover Cilium's Tetragon and its capabilities
```
Comment (michi-covalent) [inline docs/content/en/docs/overview/_index.md:7]:
```suggestion
  Discover Cilium Tetragon and its capabilities
```

File: docs/layouts/partials/footer.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <footer class="footer">
   <div class="footer__inner">
-  <p> Tetragon was originally created by <a class="" href="https://isovalent.com/" target="_blank" rel="noopener noreferrer">Isovalent</a>
+  <p> Cilium's Tetragon was originally created by <a class="" href="https://isovalent.com/" target="_blank" rel="noopener noreferrer">Isovalent</a>
```
Comment (michi-covalent) [inline docs/layouts/partials/footer.html:3]:
```suggestion
  <p> Cilium Tetragon was originally created by <a class="" href="https://isovalent.com/" target="_blank" rel="noopener noreferrer">Isovalent</a>
```

**Review by michi-covalent — COMMENTED** [review]
thanks for the fix! two minor nits based on the changes proposed in #1672. i guess we are referring to tetragon as "Cilium Tetragon".

File: docs/content/en/docs/overview/_index.md (hunk 1 above)
Comment (hacktivist123) [reply docs/content/en/docs/overview/_index.md:7]: @lizrice can you clarify this? i was a bit confused, how do we want to position this? is it Cilium's tetragon or Cilium tetragon?

Comment (lizrice) [reply docs/content/en/docs/overview/_index.md:7]: "Cilium Tetragon" is correct, so Michi's suggested changes are good

Comment (hacktivist123) [reply docs/content/en/docs/overview/_index.md]: oh thanks, I was a bit confused on if we are now calling it Cilium Tetragon or if we want to indicate that it's a part of Cilium by just saying Cilium's Tetragon.

@michi-covalent i've committed the change

**Review by michi-covalent — APPROVED** [review]
LGTM please squash these two commits 🙏

**Review by mtardy — APPROVED** [review]
Thanks for doing that!

**Pull request merged**